csharp/private/rules/nunit_test.bzl

```diff
@@ -70,6 +71,10 @@ csharp_nunit_test = rule(
             doc = "A list of files to embed in the DLL as resources.",
             allow_files = True,
         ),
+        "out": attr.string(
+            doc = "File name, without extension, of the built assembly.",
+            default = "",
```

Comment:
Not: This is the default default, so needn't be specified

Reply:
And yet, without it, CI fails, saying that there's no `out` attribute specified. What's the right answer for that, then?

Reply:
`out_file_name = name if out == None else out`

That's just because `out` was not `None`, so `out_file_name` became `""`. `out = out_file_name` was then invalid on `CSharpAssembly`.

Should be good now, with out without these explicit defaults.

Reply:
Ah, of course. I got in a hurry and didn't think it all the way through. Will fixup and revert the `default=""` on the rules, since they're unneeded.

Reply:
Fixed in subsequent commit